app/models/state.py

```diff
@@ -4,15 +4,15 @@
 class State(db.Model):
     __tablename__ = 'states'
 
-    state_code = db.Column(db.String(), primary_key=True)
-    state_name = db.Column(db.String())
+    state_fips = db.Column(db.String(), primary_key=True)
+    state_code = db.Column(db.String())
 
-    def __init__(self, state_code, state_name):
+    def __init__(self, state_fips, state_code):
         self.state_code = state_code
-        self.state_name = state_name
+        self.state_fips = state_fips
 
     def __repr__(self):
-        return 'Summary(state_code=%s, state_name=%s)' % (self.state_code, self.state_name)
+        return 'Summary(state_fips=%s, state_code=%s)' % (self.state_fips, self.state_code)
 
     def json(self):
-        return {'title': self.title, 'state': self.state_name}
\ No newline at end of file
+        return {'state fips': self.state_fips, 'state code': self.state_code}
```

Comment:
Can you add a new line at the end of the file to fix the pep8?

Reply:
I added the code for checking pep 8 problems and fixed all the possible pep 8 issues that the code found